packages/file-upload/src/director/FileAdapter.ts

```diff
@@ -28,6 +29,11 @@ export interface IFileAdapter<Context = unknown> extends IBaseAdapter {
   createDownloadStream: (file: UploadedFile, unchainedAPI: Context) => Promise<Readable>;
 }
 export const FileAdapter: Omit<IFileAdapter, 'key' | 'label' | 'version'> = {
+  signUrl() {
+    return new Promise<string | null>((resolve) => {
```

Comment:
If you throw here, only adapters that can sign url's (for private reasons for ex.) sign, and other's crash (like minio) for isPrivate files, right?